pkg/handlers/dev/start.go

```diff
@@ -35,11 +35,11 @@ func (cmd *StartCmd) Run(c *cc.CommonCtx) error {
 		if err != nil {
 			return err
 		}
-		color.Yellow("!!! topaz sidecar is already running")
+		color.Yellow("!!! topaz edge authorizer is already running")
```

Comment:
Can't we just say "topaz is already running"? The running topaz could be any instance, not just a managed edge?